libs/ledgerjs/packages/cryptoassets/package.json

```diff
@@ -33,6 +33,7 @@
     "import:cal-tokens": "ts-node ./src/crypto-assets-importer/index.ts"
   },
   "dependencies": {
+    "@ledgerhq/live-env": "workspace:^",
```

Comment:
question:
do we really want to add dependency of live-env in this package?

Or just use kind of
```
const CRYPTO_ASSETS_BASE_URL = "https://cdn.live.ledger.com/cryptoassets";
```
as before?

Reply:
Yeah I'm not sure either, it's just that if we want to test with a staging CAL for example, having it hardcoded would prevent everything from working compared to everything else in the live 🤷‍♂️

Reply:
ok, it looks good.
just need to confirm whether this cryptoassets is used by projects other that ledger live.